site/app/config/packages/old_sound_rabbit_mq.yml

```diff
@@ -30,3 +33,14 @@ old_sound_rabbit_mq:
             exchange_options: {name: 'update-project-info', type: fanout}
             queue_options:    {name: 'update-github-metadata'}
             callback:         update_github_metadata_consumer
+    multiple_consumers:
```

Comment:
Did you double-check that this leads to two queues?

Reply:
yep, it does